virtwho/config.py

```diff
@@ -759,6 +764,11 @@ class ConfigSection(collections.MutableMapping):
     __marker = object()
 
     def __init__(self, section_name, wrapper):
+        """
+        Initialization of ConfigSection instance
+        :param section_name: The name of section
+        :param wrapper: ?
```

Comment:
The intention is for the wrapper to be a reference to the containing VWEffectiveConfig object.
If we do not feel the need for it, it could be removed.